easybuild/easyconfigs/t/TCLAP/TCLAP-1.2.2-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'TCLAP'
+version = '1.2.2'
+
+homepage = 'http://tclap.sourceforge.net/'
+description = """TCLAP is a small, flexible library that provides a simple interface for defining and accessing
+command line arguments. It was intially inspired by the user friendly CLAP libary."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [('http://sourceforge.net/projects/%(namelower)s/files/', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f5013be7fcaafc69ba0ce2d1710f693f61e9c336b6292ae4f57554f59fde5837']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```

Comment:
@crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead